client/app/telecom/telephony/line/management/convert/telecom-telephony-line-management-convert.controller.js

```diff
@@ -74,13 +74,27 @@ angular.module("managerApp").controller("TelecomTelephonyLineConvertCtrl", funct
     };
 
     self.filterServices = function (services) {
-        return _.filter(services, function (service) {
-            return ["sip", "mgcp"].indexOf(service.featureType) > -1;
+        var filteredServices = _.filter(services, function (service) {
+            return !_.some(service.offers, _.method("includes", "individual"));
+        });
+
+        return _.filter(filteredServices, function (service) {
+            return ["sip", "mgcp", "voicefax"].indexOf(service.featureType) > -1;
+        });
+    };
+
+    self.filterServicesAsync = function (services) {
+        return OvhApiPackXdslVoipLine.Erika().services().aggregate("packName").execute().$promise.then(function (lines) {
+            var filteredServices = _.filter(services, function (service) {
+                return !_.some(lines, { key: service.serviceName });
+            });
+
+            return $q.when(filteredServices);
         });
     };
 
     self.getBulkParams = function () {
-        return {};
+        return { };
     };
```

Comment:
Empty object?

Reply:
it needs change on telephony-bulk-modal-controller.js to allow null getBulkParams function => it will be done :)